src/open_inwoner/accounts/templates/registration/login.html

```diff
@@ -3,7 +3,9 @@
 
 
 {% block header_image %}
+<aside aria-label="{% trans "Header afbeelding" %}">
```

Comment:
I don't think the label adds much value if the section is purely decorative. I think it's better to hide the section from asssistive technology.

 > Adding aria-hidden="true" to an element removes that element and all of its children from the accessibility tree. This can improve the experience for assistive technology users by hiding:
 >
 > Purely decorative content, such as icons or images
Duplicated content, such as repeated text
Offscreen or collapsed content, such as menus
 > 
 > The presence of the aria-hidden attribute hides content from assistive technology but doesn't visually hide anything.